VTEX - Orders API.json

```diff
@@ -4959,7 +4959,7 @@
             "get": {
                 "tags": ["User orders"],
                 "summary": "Retrieve user order details",
-                "description": "Lists all details from a user's order, through client's perspective. \r\n\r\n> Note that these requests are meant to be made by **Call center operator** profiles. Otherwise, they will return only orders from the same email making the request.",
+                "description": "Lists all details from a user's order, through client's perspective. \r\n\r\n> You can only access information from orders created in the last two years, and that same period is valid for customers through [My Account](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh). To access order data from previous years, please contact [our Support](https://support.vtex.com/hc/en-us/requests). Your access to orders placed before the last two years is temporary and will be available for 15 days. \r\n\r\n> Note that these requests are meant to be made by **Call center operator** profiles. Otherwise, they will return only orders from the same email making the request.",
```

Comment:
```suggestion
                "description": "Lists all details from an order, through the perspective of the customer who placed the order. \r\n\r\n> You can only access information from orders created in the last two years, and that same period is valid for customers through [My Account](https://help.vtex.com/en/tutorial/how-my-account-works--2BQ3GiqhqGJTXsWVuio3Xh). To access order data from previous years, please contact [our Support](https://support.vtex.com/hc/en-us/requests). Your access to orders placed before the last two years is temporary and will be available for 15 days. \r\n\r\n> Note that this request is meant to be made by **Call center operator** profiles. Otherwise, it will return only orders from the same email making the request.",
```